cap/modules/deposit/cli.py

```diff
@@ -142,6 +146,70 @@ def create_deposit(file, ana, roles, users, owner, save_errors):
     click.secho(f"\nProcess finished and record(s) added.", fg='green')
 
 
+@fixtures.command('token-generator')
+@click.option('--token-name', '-n',
+              'token_names',
+              required=True,
+              multiple=True,
+              help='name of the token')
+@click.option('--user', '-u',
+              'user_emails',
+              required=True,
+              multiple=True,
+              help='email of the registered user')
+@with_appcontext
+def token_generator(token_names, user_emails):
+    """
+    Return method to generate write token for users and save in a file.
+    Usage:
+        cap fixtures token-generator -u <user_email> -n <token_name>
+    """
+    token_file_path = current_app.config.get('TESTS_E2E_TOKEN_FILE')
+    if not len(token_names) == len(user_emails):
+        click.secho('Provide the same number of arguments.', fg='red')
+        sys.exit(1)
+    user_info = zip(token_names, user_emails)
+    tokens = {}
+
+    def _get_or_create_token(token_name, user_email):
```

Comment:
make `_get_or_create_token` standalone method (also remove `click` things from inside - they should be on the commands logic)